frontend/src/components/Cards/AlumniCard.tsx

```diff
@@ -0,0 +1,114 @@
+import React from "react";
+import { makeStyles, Theme, createStyles } from "@material-ui/core/styles";
+import Card from "@material-ui/core/Card";
+import CardMedia from "@material-ui/core/CardMedia";
+import CardContent from "@material-ui/core/CardContent";
+import Typography from "@material-ui/core/Typography";
+import IconButton from "@material-ui/core/IconButton";
+import { Email, GitHub, LinkedIn } from "@material-ui/icons";
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      width: "350px",
+      background: "transparent",
+      border: "2px solid cyan",
+      borderRadius: "24px",
+      boxShadow: "0px 0px 14px 5px #11BFD7",
+    },
```

Comment:
Avoid hard-coding values whenever possible. You can use [CSS variables](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties) instead.